src/store/reducers/describe.ts

```diff
@@ -28,21 +27,34 @@ const describe: Reducer<
             };
         }
         case FETCH_DESCRIBE.SUCCESS: {
-            let newData;
+            const data = action.data;
 
-            if (action.data.Path) {
+            const isCurrentDescribePath = data.Path === state.currentDescribePath;
+
+            // Do not reset loading state if it is not the current describe path
+            // May be needed in case of schema path change while previous fetch has not been finished
+            const wasLoaded = isCurrentDescribePath || state.wasLoaded;
```

Comment:
may be `const wasLoaded = isCurrentDescribePath && state.wasLoaded;` ?

Reply:
No, if it is not a current path, we just take the value that is in state